karma-shared.conf.js

```diff
@@ -10,8 +10,16 @@ module.exports = function(config, specificOptions) {
     browserDisconnectTimeout: 10000,
     browserDisconnectTolerance: 2,
     browserNoActivityTimeout: 30000,
-
-
+    reporters: ['spec'],
+    specReporter: {
+      maxLogLines: 5,             // limit number of lines logged per test
+      suppressErrorSummary: true, // do not print error summary
+      suppressFailed: false,      // do not print information about failed tests
```

Comment:
I guess that these two lines do not hide the actual failure information, which is crucial to understanding why the job fails?